### DISCUSSION TITLE
[Feature Request] add option/toggle to end player at video end

### DISCUSSION THREAD

**Discussion by J70P** [moneytoo/Player#535] [Ideas]
Please add an option/toggle to end the player at video end (and don't stop and display delete/play buttons).

Or at least let the display turn off after a few seconds. Don't leave the display on indefinitely and show the two buttons.

**Comment by moneytoo** [comment]
> Or at least let the display turn off after a few seconds. Don't leave the display on indefinitely and show the two buttons.

When video is not playing, screen turns off after the standard system timeout.

**Reply by J70P** [reply]
Hm ...

I just tested it again. This time the display went off as you wrote.

I've been using Just Player for maybe 2 weeks. Yesterday was at least the second time that the screen stayed on at the end of the video. I only noticed this after a certain amount of time (minutes, maybe half an hour).

I'm using the most recent F-Droid-Version (0.149) on LineageOS 20 (Android 13). Maybe your app in combination with Android 13, LineageOS, ...

(When I read your answer, I immediately thought about whether I was doing something with Tasker, depending on your app. But that's not the case.)
